.kitchen.yml

```diff
@@ -1,19 +0,0 @@
----
```

Comment:
The userdata sets up too many pre-reqs. So citadel will fail immediately, then redis becuase S3FS isn't in place.

You have to have a functional endpoint to the instance being created, which CFN does via Route53, EIP, and EC2 during build. (or the ssl creation/validation fails)

You'll need the bucket created with the special policies needed for S3FS and Citadel which use the IAM role that's assigned to the EC2 node.